### PR TITLE
Added special case for the multiplcation with a scalar.

### LINKED ISSUE #3: Multiplication of a Vector/Matrix and a Scalar only works with ".*" (times), not with "*"(mtimes).
In Matlab, the * operator (mtimes) can be used for vector/matrix multiplication and for the multiplication of a vector/matrix with a scalar. With the unc-variables, the * operator can only be used for vector/matrix multiplication, but not for the multiplication of a vector/matrix with a scalar. 

Several test cases are described in https://github.com/DionTimmermann/metas-unclib-matlab-wrapper-tests/blob/main/unit_tests_multiplication.m .

### PR DESCRIPTION
**Pull request by DionTimmermann**
This pull request fixes #3 .
Test cases are shown in https://github.com/DionTimmermann/metas-unclib-matlab-wrapper-tests/blob/main/unit_tests_multiplication.m . #

### CODE REVIEW

File: @LinProp/LinProp.m [hunk 1]
```diff
@@ -1354,6 +1354,11 @@ function display(obj)
             z = x*inv(y);
         end
         function z = mtimes(x,y)
+            if isscalar(x) || isscalar(y)
+                z = times(x, y);
+                return
```
Comment (wollmich) [inline @LinProp/LinProp.m:1359]: I don't like the `return` in the if case. I like more if you would use if else end and my code in the else case.

**Comment by wollmich** [comment]
I was not aware about that special case. I was always using `.*`.

Thanks for adding that. Please modify as well the DistProp and MCProp files. And the date in header.

**Comment by wollmich** [comment]
The same problem exists as well for `mldivide` and `mrdivide`. But this should be added as a different pull request.

**Comment by DionTimmermann** [comment]
Thank you for the quick feedback and constructive. I will make the changes in the next days.

**Comment by DionTimmermann** [comment]
I have made all the requested changes. LinProp, DistPro, and MCProp pass all the tests in https://github.com/DionTimmermann/metas-unclib-matlab-wrapper-tests/blob/main/unit_tests_multiplication.m that address multiplication with a vector.

The tests that are "failed" are differences in the implementation of times (.*), which I would consider a separate issue. There are also some "warnings" due to differences in the error messages returned by both times and mtimes. If you would like me to, I could fix these issues at a later date.

**Review by wollmich — APPROVED** [review]

**Comment by wollmich** [comment]
> The same problem exists as well for `mldivide` and `mrdivide`. But this should be added as a different pull request.

I'll take care about that.

**Pull request merged**